packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -325,6 +326,20 @@ macro_rules! impl_tx_wrapper {
             }
         }
 
+        impl TryFrom<FuelTransaction> for $wrapper {
+            type Error = Error;
+
+            fn try_from(tx: FuelTransaction) -> Result<Self> {
+                match tx {
+                    FuelTransaction::$wrapped(tx) => Ok(tx.into()),
+                    _ => Err(error!(
+                        Other,
+                        "couldn't convert Transaction into a wrapper of type $wrapper"
```

Comment:
`Err(error!(Other, "...")` feels wrong and too vague

Reply:
there is a `error_transaction!` macro with `Other`.. not great but still better then only `Other`